websites/K/Keep Talking and Nobody Explodes/presence.ts

```diff
@@ -2,158 +2,215 @@ const presence = new Presence({
     clientId: "681116862930747520"
   }),
   strings = presence.getStrings({
-    browsing: "presence.activity.browsing",
-    reading: "presence.activity.reading"
+    reading: "presence.activity.reading",
```

Comment:
`presence.activity.*` will be removed in v3, use this instead: 
```suggestion
    reading: "general.reading",
```